src/main/java/com/synopsys/integration/rest/client/IntHttpClient.java

```diff
@@ -230,7 +230,7 @@ private void addBuilderProxyInformation() {
         }
     }
 
-    private Response handleClientExecution(HttpUriRequest request) throws IntegrationException {
```

Comment:
We should determine if this makes sense outside the needs of a test - I'm never comfortable making changes to the code based on tests alone.

Reply:
Yeah I wasn't in love with that either, but generally when mocking, I want to intercept at as high a level as possible. If there is a better way for us to return a specific response here I would love to see it.